entity-store/src/main/kotlin/jetbrains/exodus/entitystore/orientdb/query/OCondition.kt

```diff
@@ -117,11 +117,21 @@ class OEdgeExistsCondition(
     }
 }
 
+class OSingleVertexCondition(
+    val rid:String
+):OCondition {
+    override fun sql(builder: StringBuilder) {
+        builder.append("@rid=?")
```

Comment:
According to the documentation:

> NOTE: using @rid in where condition slow down queries. Much better to use the [Record ID](https://orientdb.com/docs/3.2.x/datamodeling/Concepts.html#record-id) as target. Example: change this: select from Profile where @rid = #10:44 with this: select from #10:44 *

https://orientdb.com/docs/3.2.x/sql/SQL-Where.html

So for me it sounds like it's better to use "SELECT * from <rid>" and use not Condition but dedicated Select instead: `IdSelect`